control-plane-operator/controllers/hostedcontrolplane/cloud/aws/providerconfig.go

```diff
@@ -17,13 +17,15 @@ const configTemplate = `[Global]
 Zone = %s
 VPC = %s
 KubernetesClusterID = %s
-SubnetID = %s`
+SubnetID = %s
+RoleARN = %s
+WebIdentityTokenFile = %s`
```

Comment:
if we follow this approach https://github.com/kubernetes/cloud-provider-aws/pull/238 do we really need this?